qiskit_experiments/library/hamiltonian/heat_zx.py

```diff
@@ -0,0 +1,220 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+HEAT experiments for ZX Hamiltonian.
+"""
+
+from typing import Tuple, Optional
+
+import numpy as np
+from qiskit.circuit import QuantumCircuit
+from qiskit.providers import Backend
+
+from .heat_base import BatchHeatHelper, HeatElement
+from .heat_analysis import HeatAnalysis
+
+
+class ZXHeat(BatchHeatHelper):
+    """HEAT experiment for the ZX-type entangler.
+
+    # section: overview
+        This experiment is designed to amplify the error contained in the
+        ZX-type generator, a typical Hamiltonian implemented
+        by a cross resonance drive, which is a foundation of the CNOT gate.
+
+        The echo circuit refocuses ZX rotation to identity (II) then applies
+        a pi-pulse along the interrogated error axis. X error and Y error are
```

Comment:
```suggestion
        The echo circuit refocuses the ZX rotation to the identity (II) and then applies
        a pi-pulse along the interrogated error axis. X errors and Y errors are
```
Is this true? I'm confused by the circuits in the parsed literal. The echo seems to be made of single-qubit gates on the target in which case you cannot refocus ZX to the identity. Can you clarify?

Reply:
Yes, it is not too clear. What it is trying to say is that the single qubit undoes the ZX operation given that the state of the control qubit is known (the sign of the target qubit's X rotation is based on the state of the control qubit).